lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -180,6 +198,24 @@ def __init__(self, name: AnyStr, type: ParamType, required: bool = False):
         self.type = type
         self.required = required
 
+    def invalid(self, json: Dict[AnyStr, Any]) -> bool:
```

Comment:
I decided to write specific unit tests for the Schema classes, and in the process realized that I should really do some refactoring to get better encapsulation...